pkg/mmodel/balance_position.go

```
@@ -0,0 +1,153 @@
+		return pos
+	}
+
+	// Overdraft enabled but unlimited → omit the field entirely
+	// (nil pointer + omitempty). Absence on the wire = unlimited.
+	if !settings.OverdraftLimitEnabled {
+		return pos
+	}
+
+	// Overdraft enabled AND limited → headroom = limit − used.
+	var limit decimal.Decimal
+	if settings.OverdraftLimit != nil {
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (no shared variables above if)

(wsl_v5)